lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1016,12 +1018,12 @@ ClangImporter::createClangInvocation(ClangImporter *importer,
   // to missing files and report the error that clang would throw manually.
   // rdar://77516546 is tracking that the clang importer should be more
   // resilient and provide a module even if there were building it.
-  auto VFS = clang::createVFSFromCompilerInvocation(
+  auto TempVFS = clang::createVFSFromCompilerInvocation(
       *CI, *tempClangDiags,
       importer->Impl.SwiftContext.SourceMgr.getFileSystem());
```

Comment:
Shouldn't we pass in `VFS` here?

Reply:
Ah yes, indeed it should. Thanks, nice catch.